### ISSUE TITLE
Content-Type Header::parse expects string

### ISSUE THREAD

**Issue by tomcastleman** [neomerx/json-api#117]
There is an issue with parsing the Content-Type header.

According to PSR-7 `Message::getHeader()` always returns an array, therefore `$contentType` is set to an array in [ParametersParser::parse()](https://github.com/neomerx/json-api/blob/master/src/Http/Parameters/ParametersParser.php#L61).
However [Header::parse()](https://github.com/neomerx/json-api/blob/master/src/Http/Headers/Header.php#L81) is expecting a string for `$header` and consequently `preg_split` throws an exception:

```
ErrorException in /vendor/neomerx/json-api/src/Http/Headers/Header.php line 88:
preg_split() expects parameter 2 to be string, array given
```

**Comment by neomerx** [comment]
Very nicely spotted. Thanks a lot.

**Comment by neomerx** [comment]
released in [v0.7.1](https://github.com/neomerx/json-api/releases/tag/v0.7.1)
